esx_service/vmdk_ops.py

```diff
@@ -648,10 +648,13 @@ def disk_detach(vmdk_path, vm):
 
     try:
         wait_for_tasks(si, [vm.ReconfigVM_Task(spec=spec)])
-    except vim.fault.GenericVmConfigFault as ex:
-        for f in ex.faultMessage:
-            logging.warning(f.message)
-        return err("Failed to detach " + vmdk_path)
+    except Exception as ex:
```

Comment:
Why not just catch & handle "vim.Fault.VimFault" exception and pass others to caller? This would eliminate `if isinstance(...)`